pkg/apis/batch/validation/validation.go

```diff
@@ -310,6 +311,9 @@ func ValidateCronJobSpec(spec *batch.CronJobSpec, fldPath *field.Path, opts apiv
 	} else {
 		allErrs = append(allErrs, validateScheduleFormat(spec.Schedule, fldPath.Child("schedule"))...)
 	}
+	if len(spec.TimeZone) > 0 {
+		allErrs = append(allErrs, validateTimeZone(spec.TimeZone, fldPath.Child("timeZone"))...)
+	}
```

Comment:
can we prevent use of both the API field and the unsupported/deprecated embedded TZ or CRON_TZ? that would make honoring the API field way simpler

Reply:
As mentioned elsewhere, we'll fail validation when both are present, suggesting to use only `.spec.timeZone`